setup.py

```diff
@@ -6,7 +6,7 @@
 # Get the long description from the README file
 LONG_DESCRIPTION = (here / "README.md").read_text(encoding="utf-8")
 
-VERSION = "0.1.22"
+VERSION = "0.1.23"
```

Comment:
I don't think you need to change this, pretty sure the workflow will do it. It's been awhile since I set it up, but looking back on the recent PRs that merged they didn't edit the version.

Reply:
Reverted this change.